api/applications_api.py

```diff
@@ -203,6 +203,8 @@ class ApplicationEventSerializer(serializers.ModelSerializer):
         help_text="List of reservation units applied for this event with priority included.",
     )
 
+    status = serializers.CharField(help_text="Status of this application")
```

Comment:
Should this be 'Status of applicationEvent'? 

It would also be nice from API users perspective to have some additional documentation such as the valid values and when to use which. Currently it is documented (in swagger) as:

"status* | string Status of this application"

Additionally I think the error message could also be improved. For example now I have a broken UI at hand and all the api tells me is '["Invalid status"] 

Does this refer to status of the application or applicationEvent?

Reply:
Good spots :+1:  
Yes, this considers the ApplicationEvent. 
I'll make PR for fixing these issues